src/sonic-yang-models/yang-models/sonic-vlan.yang

```diff
@@ -90,6 +90,12 @@ module sonic-vlan {
 					}
 				}
 
+				leaf static_anycast_gateway {
+					description "Enable/disable static anycast gateway for the vlan interface";
+					type boolean;
+					default false;
+        }
```

Comment:
>         [](http://example.com/codeflow?start=0&length=8)

Mixing spaces and tabs in this file.

Reply:
Already unified indentation to spaces and aligned format